arcgis-ios-sdk-samples/Augmented reality/Explore scenes in flyover AR/README.md

```diff
@@ -38,7 +38,7 @@ This sample requires a device that is compatible with ARKit 1.0 on iOS.
 
 **Flyover AR** is one of three main patterns for working with geographic information in augmented reality. See the [guide doc]() for more information.
 
-This sample uses the ArcGIS Runtime Toolkit. See [Display scenes in augmented reality](https://developers.arcgis.com/ios/latest/swift/guide/display-scenes-in-augmented-reality.htm) in the guide to learn about the toolkit and how to add it to your app.
+This sample uses the ArcGIS Runtime Toolkit. See [Augmented reality](https://developers.arcgis.com/ios/latest/swift/guide/display-scenes-in-augmented-reality.htm) in the guide to learn about the toolkit and how to add it to your app.
```

Comment:
I'm not sure why the standardized README has this wrong. The convention for introducing links that adhere to accessibility standards is to cite the topic or section title as the linking text. So it should be stated like: 

```suggestion
This sample uses the ArcGIS Runtime Toolkit. See the section [Enable your app for AR using AR Toolkit](https://developers.arcgis.com/ios/latest/swift/guide/display-scenes-in-augmented-reality.htm) in the iOS Guide to learn about the toolkit and how to add it to your app.
```